### PR TITLE
Material redesign

### PR DESCRIPTION
**Pull request by PranavPurwar**
Add material you for ManageLibraryActivity, DesignDrawer, and some improvements in DesignActivity

### CODE REVIEW

File: .idea/intellij-javadocs-4.0.1.xml
Comment (ilyassesalama) [inline .idea/intellij-javadocs-4.0.1.xml:1]: Why you didn't exclude these files? I thought they're ignored.

**Comment by JavkhlanK** [comment]
merged most of this in manually, but I think the rest in like ManageLibraryActivity or the compile log screen will be done when those activities will be fully made Material You

**Comment by PranavPurwar** [comment]
> merged most of this in manually, but I think the rest in like ManageLibraryActivity or the compile log screen will be done when those activities will be fully made Material You

I thought I made ManageLibraryActivity material you in this pr??? the sub sections are different activities

**Comment by JavkhlanK** [comment]
I said most. The activity will probably get a redesign to be like Local library Manager anyway, so those few M3 adjustments won't be needed anyway then.

File: app/src/main/java/com/besome/sketch/editor/view/ViewEditor.java [hunk 1]
```diff
@@ -337,7 +337,6 @@ public boolean onTouch(View view, MotionEvent motionEvent) {
                     boolean z = false;
                     for (int i3 = 0; i3 < uyVar.getData().size(); i3++) {
                         ViewBean viewBean = uyVar.getData().get(i3);
-                        if (c(viewBean)) {
```
Comment (hasrat-ali) [inline app/src/main/java/com/besome/sketch/editor/view/ViewEditor.java:340]: @PranavPurwar explanation?

Comment (JavkhlanK) [reply app/src/main/java/com/besome/sketch/editor/view/ViewEditor.java:340]: look at the rest of the commit, the entire method used to do nothing except return `true`

**Pull request closed without merging**